packages/react-instantsearch-dom/src/index.js

```diff
@@ -66,6 +66,10 @@ export { default as Stats } from './widgets/Stats';
 export { default as ToggleRefinement } from './widgets/ToggleRefinement';
 export { default as VoiceSearch } from './widgets/VoiceSearch';
 export { default as QueryRuleCustomData } from './widgets/QueryRuleCustomData';
+export { default as EXPERIMENTAL_Answers } from './widgets/Answers';
+
+// hooks
+export { default as EXPERIMENTAL_useAnswers } from './hooks/useAnswers';
```

Comment:
I think hooks can't start with anything else than `use`, or is all-caps prefix allowed?